Snowflake.Data.Tests/IntegrationTests/SFDbCommandIT.cs

```diff
@@ -732,49 +731,71 @@ static void ArrayBindTest(string connstr, string tableName, int size)
                     //cmd.ExecuteNonQuery();
 
                 }
-                System.Threading.Thread.Sleep(2000); // wait for 2 seconds, in case other test still running.
+                //System.Threading.Thread.Sleep(2000); // wait for 2 seconds, in case other test still running.
                 conn.Close();
             }
         }
 
         [Test]
         public void TestPutArrayBindAsyncMultiThreading()
         {
-            var connStr = ConnectionString + "";
-            Thread t1 = new Thread(() => ThreadProcess1(ConnectionString));
-            Thread t2 = new Thread(() => ThreadProcess2(ConnectionString));
+            var t1TableName = TableName + 1;
+            var t2TableName = TableName + 2;
+            CreateOrReplaceTable(t1TableName, new []
```

Comment:
Same table struct we have 3 times.
if CreateOrReplaceTable has DbConnection it could remain in ArrayBindTest.

Reply:
Unfortunately, it cannot reside in `ArrayBindTest` as this one is static whereas `CreateOrReplaceTable` is non-static.

Reply:
Or better - change all static to private.

Reply:
Done.